go-controller/pkg/node/management-port.go

```diff
@@ -23,16 +32,42 @@ func createManagementPort(nodeNameOriginal string, hostSubnets []*net.IPNet, nod
 	// will try to fetch and what kubernetes provides, thus failing to
 	// create the port on the logical switch.
 	// Until the above is changed, switch to a lowercase hostname
-	nodeName := strings.ToLower(nodeNameOriginal)
+	nodeName = strings.ToLower(nodeName)
 
+	switch config.OvnKubeNode.Mode {
+	default:
+		return &managementPort{
+			nodeName:    nodeName,
+			hostSubnets: hostSubnets,
+		}
+	case types.NodeModeSmartNIC:
+		return &managementPortSmartNIC{
+			nodeName:    nodeName,
+			hostSubnets: hostSubnets,
+			vfRepName:   config.OvnKubeNode.MgmtPortNetdev,
+		}
+	case types.NodeModeSmartNICHost:
+		return &managementPortSmartNICHost{
+			hostSubnets: hostSubnets,
+			netdevName:  config.OvnKubeNode.MgmtPortNetdev,
```

Comment:
In theory, this allows any VF name be specified and act as the backend of ovn-k8s-mp0, not just vf0, right? 
Does it matter if different VF is used after node reboot?

Reply:
> In theory, this allows any VF name be specified and act as the backend of ovn-k8s-mp0, not just vf0, right?

That is correct

> Does it matter if different VF is used after node reboot?

for ovnkube node it doesnt as at that point in time it would not have an ovn-k8s-mp0 netdev.  However the VF which is associated with this netdev must not be allocated by device plugin.